File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/TokenResponse.java
package com.lattis.ellipse.data.network.model.response;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 4/25/17.
 */

public class TokenResponse {

    @SerializedName("rest_token")
    public String restToken;
    @SerializedName("refresh_token")
    public String refreshToken;

    public String getRestToken() {
        return restToken;
    }

    public String getRefreshToken() {
        return refreshToken;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/authentication/TokenResponse.kt
package com.lattis.data.entity.response.authentication

import com.google.gson.annotations.SerializedName

/**
 * Created by ssd3 on 4/25/17.
 */

class TokenResponse {

    @SerializedName("rest_token")
    var restToken: String? = null
    @SerializedName("refresh_token")
    var refreshToken: String? = null
}
